app/routes/network/$networkName.tsx

```diff
@@ -20,7 +20,10 @@ export const loader: LoaderFunction = async () => {
 export default function Page() {
   const { networkStatuses } = useLoaderData<DynamicNetworkDetailPageProps>()
   const params = useParams()
-  const networkName = params.networkName as string
+
+  if (!params.networkName) throw new Error("Missing network name")
```

Comment:
Do we need to throw here? If there is no network name we can assume it's a req for a different page, and this route doesn't get hit?

Reply:
I suggested this in another PR to avoid the type assertion, but I'm good with keeping it as it was